### PR TITLE
chore(fixtures): remove extraneous ballot package files

### PR DESCRIPTION
**Pull request by eventualbuddha**
## Overview
Now that we're no longer producing or importing ballot PDFs and layouts in ballot packages, this commit removes the ballot package fixtures.

Refs #3374, follow-up to #3380.

## Demo Video or Screenshot
n/a

## Testing Plan
Automated.

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/types/src/ballot_package.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  precinctId: PrecinctId;
+}
+
+export interface BallotConfig extends BallotStyleData {
+  filename: string;
+  layoutFilename: string;
+  locales: BallotLocale;
+  isLiveMode: boolean;
+  isAbsentee: boolean;
+}
```
Comment (adghayes) [inline libs/types/src/ballot_package.ts]: Can we get rid of this too?

File: apps/central-scan/backend/src/central_scanner_app.ts [hunk 2]
```diff
@@ -126,10 +122,17 @@ function buildApi({
 
     /* istanbul ignore next - only used by Cypress */
     async configureWithSampleBallotPackageForIntegrationTest(): Promise<void> {
-      const fileContents = await fs.readFile(
-        '../integration-testing/cypress/fixtures/ballot-package.zip'
+      if (process.env.CI !== 'true') {
+        throw new Error(
+          'This function is only intended to be used in CI environments'
```
Comment (adghayes) [inline apps/central-scan/backend/src/central_scanner_app.ts:127]: Why?

**Review by adghayes — APPROVED** [review]
See my two questions, but otherwise LGTM.

File: apps/central-scan/backend/src/central_scanner_app.ts (hunk 2 above)
Comment (eventualbuddha) [reply apps/central-scan/backend/src/central_scanner_app.ts:127]: I guess we do run it sometimes locally. Mostly I was trying to disallow this in production.

File: libs/types/src/ballot_package.ts (hunk 1 above)
Comment (eventualbuddha) [reply libs/types/src/ballot_package.ts]: Done.

**Pull request merged**